src/bootstrap/src/core/config/config.rs

```diff
@@ -2104,9 +2104,6 @@ impl Config {
         match download_ci_llvm {
             None => self.channel == "dev" && llvm::is_ci_llvm_available(&self, asserts),
             Some(StringOrBool::Bool(b)) => b,
-            Some(StringOrBool::String(s)) if s == "if-available" => {
```

Comment:
This needs a transition period, otherwise switching between branches before this and after this will be painful (since config.toml can't be compatible with both).

Reply:
if you just set it to if-unchanged, it should work just fine